resources/api.rb

```diff
@@ -2,8 +2,13 @@
 property :content, String, default: ''.freeze
 property :args, [Hash, String, NilClass], desired_state: false
 property :endpoint, String, desired_state: false, identity: true, default: node['nexus3']['api']['endpoint']
-property :username, String, desired_state: false, identity: true, default: 'admin'.freeze
-property :password, String, desired_state: false, identity: true, sensitive: true, default: 'admin123'.freeze
+property :username, String, desired_state: false, identity: true, default: node['nexus3']['api']['username']
```

Comment:
Why changing the formatting of the properties?

Reply:
To avoid assumptions about default values when you have multiple 'nexus3' resources with different values in their attributes, which you need to pass to the properties. I haven't found a way to call:

```
nexus3 'foo' do
  property blah
  nexus3_api property blah
end
```
i.e., set the properties for downstream resources from the main calling resource.

If you meant the actual code layout in the editor, it is to respect max 120 chars per line.